README.md

```diff
@@ -101,6 +101,42 @@ The crates included as part of Tracing are:
 [sub-crates]: https://crates.io/crates/tracing-subscriber
 [sub-docs]: https://docs.rs/tracing-subscriber
 
+## Related Crates
+
+In addition to this repository, here are also several third-party crates which
+are not maintained by the `tokio` project. These include:
+
+- [`tracing-timing`] implements inter-event timing metrics on top of `tracing`.
+  It provides a subscriber that records the time elapsed between pairs of
+  `tracing` events and generates histograms.
+- [`tracing-opentelemetry`] provides a subscriber for emitting traces to
+  [OpenTelemetry]-compatible distributed tracing systems.
+- [`tracing-honeycomb`] implements a subscriber for reporting traces to
+  [honeycomb.io].
+- [`tracing-actix`] provides `tracing` integration for the `actix` actor
+  framework.
+- [`tracing-gelf`] implements a subscriber for exporting traces in Greylog
+  GELF format.
+- [`tracing-coz`] provides integration with the [coz] causal profiler
+  (Linux-only).
+
+(if you're the maintainer of a `tracing` ecosystem crate not in this list,
+please let us know!)
```

Comment:
- Capitalize "if"?
- How about "...please let us know! We'd be happy to add your project to this list."